ModuleTest/src/test/java/com/test/TestDemo.java

```diff
@@ -0,0 +1,33 @@
+package com.test;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class TestDemo {
+
+	public static void main(String[] args) throws InterruptedException {
+		
+		System.setProperty("webdriver.chrome.driver", "F:\\Drivers\\chromedriver-win64\\chromedriver.exe");
+		
+		ChromeOptions options = new ChromeOptions();
+		options.addArguments("--remote-allow-origins=*");
+		WebDriver driver = new ChromeDriver(options);
+		
+		driver.get("https://www.youtube.com");
+		
+		Thread.sleep(3000);
+		
+		driver.findElement(By.xpath("//input[@placeholder=\"Search\"]")).sendKeys("Titli udi ud na saki");
+		
+		Thread.sleep(3000);
+		
+		driver.findElement(By.xpath("//button[@id=\"search-icon-legacy\"]")).click();
+		
+		Thread.sleep(3000);
+		
+		driver.findElement(By.xpath("//yt-formatted-string[@class=\"style-scope ytd-video-renderer\"]")).click();
+	}
```

Comment:
Please add the syso statement at the end to display whether the test case got passed or failed,